phpunit

```diff
@@ -51,11 +54,13 @@ if ($phpIniMatrix) {
 
     $phpDir = dirname(`where.exe php`);
 
-    $newCmd = '(SET X=0';
+    $newCmd = 'cmd /v:on /d /c "(SET X=0';
     foreach ($phpIniMatrix as $iniFile) {
-        $newCmd .= " & copy /Y $phpDir\\$iniFile $phpDir\\php.ini & echo. & echo Running tests with $iniFile: & $cmd & (if %%errorlevel%% NEQ 0 SET X=1)";
+        $newCmd .= " & copy /Y $phpDir\\$iniFile $phpDir\\php.ini & echo. & echo Running tests with $iniFile: & ($cmd || SET X=1)";
```

Comment:
`&` is the bash equivalent of `;`, which is what we want: a failure in a matrix line should not prevent the second line from running